culture/trust_levels.md

```diff
@@ -0,0 +1,108 @@
+This post proposes a system of **trust levels** for SourceCred the
+system, which describe how we intend to handle the very real questions
+of adversarial actors and gaming resistance over different time scales.
+
+The ideas proposed herein have been discussed informally within our
+community, but with somewhat confusingly varying semantics. It may help
+to have this shared reference to point to.
+
+**Note:** The term *trust levels* as used in this post is largely
+separate from [Discourse trust levels][discourse-trust], which are a
+construct of our forum software and not related to the SourceCred
+protocol in any way.
+
+[discourse-trust]: https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/
+
+# Motivation
+
+To achieve its goals when applied to real-world projects, SourceCred
+will need to be resistant to gaming by people who explicitly (but
+perhaps covertly) attempt to maximize their short-term or long-term
+profits at all other costs. This is because SourceCred is intended to be
+used to allocate scarce resources, and so actors will have incentives to
+acquire those resources and may have no natural incentive to contribute
+back to the community or behave in accordance with our preferred norms.
+
+However, full resistance to adversarial behavior will require
+substantial research, design, and implementation effort. Some of this
+effort cannot even begin until we have a fairly stable understanding of
+other parts of the pipeline. It doesn’t make sense to discuss defenses
+until we’ve defined the protocol in which everyone is to operate.
```

Comment:
For the layperson reading, they probably don't know what the "pipeline" is referring to. And what constitutes a stable understanding of it? I think what you are simply saying is:

> However, full resistance to adversarial behavior will require substantial research, design, and implementation effort. Some of this effort cannot even begin until the SourceCred algorithm and associated tooling has gone through substantial testing.

or something like that?